### DISCUSSION TITLE
Option to delete series

### DISCUSSION THREAD

**Discussion by kamil7x** [forem/forem#16434] [Icebox]
Hi,

I created a series, but I accidentally made a typo in name, so I created a new one and added posts to it. Dashboard series' counter shows properly only 1 series and does not display empty one. But I still have it on series selector in editor. I would like to remove it completely.

**Comment by kamil7x** [comment]
Yeah, so it looks like it's not going to happen? Any comments, why?

**Reply by jeremyf** [reply]
Oof, sorry for the long delay in a response.  This request threads the needle of a bug and a feature.  I definitely see that there is an expectation of "I'd like to be able to delete an empty series" so it doesn't show up.

However, the lack of response doesn't mean "it's not going to happen."  It means we haven't yet responded.

I am seeking clarity on both the feature and an "any comments" reply.  I don't have a timeframe for that.

**Reply by kamil7x** [reply]
Yeah, sorry for being rude.

**Comment by hunghvu** [comment]
Empty series are shown in the "Create Post" section. However, they are not shown in the dashboard as implemented in #9658. To me, this does not solve the root of an issue. The unused data is still existing in the database and it is just hidden from the users.

There should be a way to fully manage the _series_ (e.g. rename, delete, add, analytics, etc.), preferably in a separate dashboard page. We are lacking the whole set of features, so this shouldn't be considered a bug in my opinion.

![image](https://user-images.githubusercontent.com/62678592/160210493-2c09e003-a6c5-4829-bd15-cbfdc105e4cb.png)

![image](https://user-images.githubusercontent.com/62678592/160210864-b0ed457e-fb90-4046-bd7a-a0606ae57dc4.png)

**Reply by kamil7x** [reply]
I agree. Full series management would be the best here.

**Reply by kamil7x** [reply]
Including reordering posts in series

**Comment by JennieOcken** [comment]
There are a lot of requests around making our series functionality more robust.  I want to group them all together here and say we will take them as a set at some point in the future.

[Write description](https://github.com/forem/forem/discussions/15474)
[Edit](https://github.com/forem/forem/discussions/15673)
[Delete](https://github.com/forem/forem/discussions/16434)
[Embed](https://github.com/forem/forem/discussions/16805)
[Multiple Authors](https://github.com/forem/forem/discussions/15320)
[Reorder](https://github.com/forem/forem/discussions/15391)
